pydatastructs/graphs/algorithms.py

```diff
@@ -548,3 +548,32 @@ def strongly_connected_components(graph, algorithm):
         "isn't implemented for finding strongly connected components."
         %(algorithm, graph._impl))
     return getattr(algorithms, func)(graph)
+
+
+def random_walk(G, start, path_length, alpha = None):
+    import random
+    random.seed(44)
+    
+    if start:
+        path = [start]
+    else:
+        path = [random.choices(G.vertices)]
+  
+
+    for i in range(0, path_length):
+        cur = path[-1]
+        degree = len(G.neighbors(cur))
+        probablity_list = degree*[1/degree]
```

Comment:
`int`*`list`?

Reply:
Yes. If the list is of length equal to degree and the all elements in it are 1/degree.